pepys_admin/maintenance/column_data.py

```diff
@@ -404,6 +404,12 @@ def convert_column_data_to_edit_data(column_data, set_percentage=None):
             # Don't allow to edit ID columns
             continue
 
+        if key == "wargame participations":
+            # Don't allow to edit wargame participations, as joining the data
```

Comment:
@robintw - do we need to explain to analysts that we don't allow this table to be edited, but they can make changes via the `Maintain Tasks` tab?

Reply:
We could do, but to be honest I doubt they'd try and edit that manually... The 'obvious' place to change things would be in the Tasks GUI.